deployment/network-operator/templates/operator.yaml

```diff
@@ -19,6 +19,9 @@ metadata:
   name: {{ include "network-operator.fullname" . }}
   labels:
     {{- include "network-operator.labels" . | nindent 4 }}
+    {{- if .Values.operator.admissionController.enabled }}
+    control-plane:  {{ .Release.Name }}-controller
```

Comment:
we can always add this one. WDYT ?

Reply:
done

Reply:
thx. @rollandf could you also  remove the extra space ? (there are 2 by accident)